.github/PULL_REQUEST_TEMPLATE

```diff
@@ -37,3 +37,6 @@
 
 
 **Testing instructions:**
+
+
+**Proposed commit message: (wrap lines at 72 characters)**
```

Comment:
Add a comment to mention the rough structure of commit messages (which is stated in the oss-generic documentation):

```html
<!-- 
    See this link for more info on how to write a good commit message:

    https://oss-generic.github.io/process/docs/FormatsAndConventions.html#commit-message 
-->
```

Reply:
Also actually, some of the existing questions in the template could be used to craft the commit message.

In fact there will likely be a repeat of content across the existing questions and the commit message itself. However, I am not sure whether it is a good idea to remove the questions, because it serves as additional guidance for crafting good commit messages.

So, instead of removing the questions, it would be sufficient if we just put this in:

```html
<!--
    Some of the responses that you gave to the previous questions might
    provide you with the information needed to craft your commit message.
-->
```